fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -2074,4 +2101,194 @@ private void populateCodesSubSegment(StringBuilder whereClauseSegment, Modifier
         log.exiting(CLASSNAME, METHODNAME);
     }
 
+    private SqlQueryData processProfileParm(Class<?> resourceType, QueryParameter queryParm, String tableAlias)
```

Comment:
Isn't the old query builder going away completely with this change?

Reply:
Lee opened issue #2522 for this.